core/vm/jump_table.go

```diff
@@ -576,6 +576,14 @@ func newFrontierInstructionSet() JumpTable {
 			minStack:    minStack(0, 0),
 			maxStack:    maxStack(0, 0),
 		},
+		MCOPY: {
```

Comment:
You've added it to `newFrontierInstructionSet` which would activate it from genesis... (may be fine for a prototype)

Reply:
gotcha. i'm guessing it should move to `core/vm/eips.go` then?

Reply:
please see https://github.com/ethereum/go-ethereum/pull/26181/commits/cc503cfc99cf4eae307845246efd89cd01cd27a2